pkg/v1/cli/signals/signals.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
```

Comment:
This package looks like generally useful outside of the CLI. Should this be in `pkg/v1` instead of `pkg/v1/cli`?

Reply:
Do you think there will be a need for signal handling outside of Tanzu CLI or its plugins? I couldn't think of a need, but maybe you can. If so, where would you recommend I put this package?

Reply:
I can envision needing signal handling if we build a "construction kit" for controllers, but that's far into the future, so let's just keep it here.